lib/std/os/windows/user32.zig

```diff
@@ -1,104 +1,100 @@
+    KEYDOWN = 0x0100,
+    KEYUP = 0x0101,
+    CHAR = 0x0102,
+    DEADCHAR = 0x0103,
+    SYSKEYDOWN = 0x0104,
+    SYSKEYUP = 0x0105,
+    SYSCHAR = 0x0106,
+    SYSDEADCHAR = 0x0107,
+    KEYLAST = 0x0109,
+
+    COMMAND = 0x0111,
+    SYSCOMMAND = 0x0112,
+    TIMER = 0x0113,
+
+    MOUSEMOVE = 0x0200,
+    LBUTTONDOWN = 0x0201,
+    LBUTTONUP = 0x0202,
+    LBUTTONDBLCLK = 0x0203,
+    RBUTTONDOWN = 0x0204,
+    RBUTTONUP = 0x0205,
+    RBUTTONDBLCLK = 0x0206,
+    MBUTTONDOWN = 0x0207,
+    MBUTTONUP = 0x0208,
+    MBUTTONDBLCLK = 0x0209,
+    MOUSEWHEEL = 0x020A,
+    XBUTTONDOWN = 0x020B,
+    XBUTTONUP = 0x020C,
+    XBUTTONDBLCLK = 0x020D,
+};
+
+pub const WA = enum(u16) {
+    INACTIVE = 0,
+    ACTIVE = 0x0006,
+    ACTIVATE = 0x0006,
+};
+
+pub const SW = enum(u16) {
+    HIDE = 0,
+    SHOW = 5,
+};
+
 pub const PM_REMOVE = 0x0001;
 pub const PM_NOREMOVE = 0x0000;
 pub const PM_NOYIELD = 0x0002;
 
-// WM
-pub const WM_NULL = 0x0000;
-pub const WM_CREATE = 0x0001;
-pub const WM_DESTROY = 0x0002;
-pub const WM_MOVE = 0x0003;
-pub const WM_SIZE = 0x0005;
-
-pub const WM_ACTIVATE = 0x0006;
-pub const WM_PAINT = 0x000F;
-pub const WM_CLOSE = 0x0010;
-pub const WM_QUIT = 0x0012;
-pub const WM_SETFOCUS = 0x0007;
-
-pub const WM_KILLFOCUS = 0x0008;
-pub const WM_ENABLE = 0x000A;
-pub const WM_SETREDRAW = 0x000B;
-
-pub const WM_SYSCOLORCHANGE = 0x0015;
-pub const WM_SHOWWINDOW = 0x0018;
-
-pub const WM_WINDOWPOSCHANGING = 0x0046;
-pub const WM_WINDOWPOSCHANGED = 0x0047;
-pub const WM_POWER = 0x0048;
-
-pub const WM_CONTEXTMENU = 0x007B;
-pub const WM_STYLECHANGING = 0x007C;
-pub const WM_STYLECHANGED = 0x007D;
-pub const WM_DISPLAYCHANGE = 0x007E;
-pub const WM_GETICON = 0x007F;
-pub const WM_SETICON = 0x0080;
-
-pub const WM_INPUT_DEVICE_CHANGE = 0x00fe;
-pub const WM_INPUT = 0x00FF;
-pub const WM_KEYFIRST = 0x0100;
-pub const WM_KEYDOWN = 0x0100;
-pub const WM_KEYUP = 0x0101;
-pub const WM_CHAR = 0x0102;
-pub const WM_DEADCHAR = 0x0103;
-pub const WM_SYSKEYDOWN = 0x0104;
-pub const WM_SYSKEYUP = 0x0105;
-pub const WM_SYSCHAR = 0x0106;
-pub const WM_SYSDEADCHAR = 0x0107;
-pub const WM_UNICHAR = 0x0109;
-pub const WM_KEYLAST = 0x0109;
-
-pub const WM_COMMAND = 0x0111;
-pub const WM_SYSCOMMAND = 0x0112;
-pub const WM_TIMER = 0x0113;
-
-pub const WM_MOUSEFIRST = 0x0200;
-pub const WM_MOUSEMOVE = 0x0200;
-pub const WM_LBUTTONDOWN = 0x0201;
-pub const WM_LBUTTONUP = 0x0202;
-pub const WM_LBUTTONDBLCLK = 0x0203;
-pub const WM_RBUTTONDOWN = 0x0204;
-pub const WM_RBUTTONUP = 0x0205;
-pub const WM_RBUTTONDBLCLK = 0x0206;
-pub const WM_MBUTTONDOWN = 0x0207;
-pub const WM_MBUTTONUP = 0x0208;
-pub const WM_MBUTTONDBLCLK = 0x0209;
-pub const WM_MOUSEWHEEL = 0x020A;
-pub const WM_XBUTTONDOWN = 0x020B;
-pub const WM_XBUTTONUP = 0x020C;
-pub const WM_XBUTTONDBLCLK = 0x020D;
-
-// WA
-pub const WA_INACTIVE = 0;
-pub const WA_ACTIVE = 0x0006;
-pub const WM_ACTIVATE = 0x0006;
-
-// WS
+pub const CS_HREDRAW = 0x0002;
+pub const CS_VREDRAW = 0x0001;
```

Comment:
Could you add all the constants? https://docs.microsoft.com/en-us/windows/win32/winmsg/window-class-styles